worlds/celeste64/__init__.py

```diff
@@ -51,9 +57,48 @@ def create_item(self, name: str) -> Celeste64Item:
     def create_items(self) -> None:
         item_pool: List[Celeste64Item] = []
 
-        item_pool += [self.create_item(name) for name in item_data_table.keys()]
+        location_count: int = 30
+
+        if self.options.friendsanity:
+            location_count += 9
+
+        if self.options.signsanity:
+            location_count += 5
+
+        if self.options.carsanity:
+            location_count += 2
+
+        item_pool += [self.create_item(name)
+                      for name in unlockable_item_data_table.keys()
+                      if name not in self.options.start_inventory]
+
+        if self.options.move_shuffle:
+            move_items_for_itempool: List[str] = deepcopy(list(move_item_data_table.keys()))
+
+            if self.options.logic_difficulty == "standard":
```

Comment:
I think explaining this behavior somewhere would be a good idea. Like in the option description for `logic_difficulty`

Reply:
LGTM